types/applications/application.d.ts

```diff
@@ -428,7 +428,7 @@ declare namespace Application {
     /**
      * @defaultValue `[]`
      */
-    dragDrop: DragDrop[];
+    dragDrop: Partial<DragDrop>[];
```

Comment:
I noticed the same problem when I started on the other branch yesterday. But I did not have the time yesterday to check where it is actually used. If those are indeed just constructor options, instead of the actual objects, I can easily change that.